account-kit/react/src/icons/auth-icons/twitch.tsx

```diff
@@ -0,0 +1,19 @@
+import type { SVGProps } from "react";
+
+export const TwitchIcon = (
```

Comment:
nit: should we have an icon that is filled in w/ white instead of being transparent so it looks a little nicer in dark mode?

how this icon looks:

![Screenshot 2025-03-12 at 5 21 57 PM](https://github.com/user-attachments/assets/479847e0-3ca0-4914-b329-8989a20efd5b)

example from twitch's website in dark mode:

![Screenshot 2025-03-12 at 5 21 48 PM](https://github.com/user-attachments/assets/8475ce68-9a03-4e1a-9db0-4660d1be5eb2)

Reply:
Looks like this now:
<img width="439" alt="image" src="https://github.com/user-attachments/assets/5e6f5c25-5500-4d83-af9e-b97cd377727b" />